backend/molgenis-emx2-io/src/test/java/org/molgenis/emx2/io/TestDatabaseFactory.java

```diff
@@ -0,0 +1,15 @@
+package org.molgenis.emx2.io;
+
+import org.molgenis.emx2.Database;
+import org.molgenis.emx2.sql.SqlDatabase;
+
+public class TestDatabaseFactory {
+  private static Database db;
+
+  public static Database getTestDatabase() {
+    if (db == null) {
+      db = new SqlDatabase(false);
+    }
+    return db;
+  }
+}
```

Comment:
duplicate code ( graphql folder)

Reply:
dat is by design. Ik zou het kunnen centraliseren maar dan heb je kans dat we daar op gegeven moment spijt van krijgen.

Reply:
waarom is char voor char zelfde toch, kunnen altijd later spliten als nodi is